include/graphblas/algorithms/conjugate_gradient.hpp

```diff
@@ -85,7 +85,7 @@ namespace grb {
 		 * Additional outputs (besides \a x):
 		 *
 		 * @param[out]    iterations     The number of iterations the algorithm has
-		 *                               performed.
+		 *                               started.
```

Comment:
Just a comment, if it is `out`, then is is number of iterations _performed_, is that right?

Reply:
If SUCCESS were returned yes -- otherwise, it is possible that an iteration is stopped mid-way, hence the change